cdap-elastic/src/main/java/co/cask/cdap/metadata/elastic/MetadataDocument.java

```diff
@@ -85,7 +93,9 @@ public boolean equals(Object o) {
       return false;
     }
     MetadataDocument that = (MetadataDocument) o;
-    return Objects.equals(entity, that.entity) &&
+    return created == that.created &&
```

Comment:
`created` is of type Long, need to use `equals()` here for check.

Reply:
good catch! I regenerated this with IDEA, not sure what went wrong, probably I messed up.

Reply:
regenerated both hashCode and equals